wordpress-mobile/GutenbergKit
# Use Android Module

That will allow us to publish the library for Android.

To Test:
1. Run `make local-android-library`
2. Note that there's a build product at `~/.m2/repository/org/wordpress/gutenbergkit/android/unspecified/android-unspecified.aar`
3. Rename that file to end with `.zip`, then extract it
4. Open that folder, note that the assets are present

~~We'll add automatic publishing for each PR commit in a subsequent PR~~ Thanks to @mokagio, this PR now publishes the library too.
